### DISCUSSION TITLE
Content not updated in dev mode

### DISCUSSION THREAD

**Discussion by oripka** [nuxt/content#1563] [Q&A] [answered]
Just sharing... I had an issue where updated content was not reflected in the navigation in dev mode (`npx nuxi dev`). The navigation would only update after running `npx nuxi build`. Troubleshooting this I found that deleting `.nuxt` and `.output` solved the issue.

**Comment by oripka** [comment, marked as answer]
Remove `.nuxt` and `.output` to fix the problem.

**Reply by ayZagen** [reply]
I face this issue but whatever i tried didn't solve it. somehow old contents are still displayed. I tried that and additionally i tried `nuxi cleanup`, removing node_modules, running pnpm install etc. still the old content is served. Is there any other directory that nuxt uses to store rendered files ?

**Reply by oripka** [reply]
maybe related to this https://github.com/nuxt/content/issues/2100#issuecomment-1864748213 (tl;dr refresh with browser dev tools open)

**Reply by ayZagen** [reply]
@oripka thanks for the quick answer, I always keep my devtools open but i tried even clearing site data still nothing helps. Also tried while cache is disabled `Network > Disable Cache`. I am eating my nails rn.
